test/facility-activation-transaction/get-facility-activation-transaction.api-test.ts

```diff
@@ -0,0 +1,167 @@
+  });
+
+  withClientAuthenticationTests({
+    givenTheRequestWouldOtherwiseSucceed: () => {
+      givenAuthenticationWithTheIdpSucceeds();
+      givenRequestToGetFacilityActivationTransactionInAcbsSucceeds();
+    },
+    makeRequestWithoutAuth: (incorrectAuth?: IncorrectAuthArg) =>
+      api.getWithoutAuth(getFacilityActivationTransactionUrl, incorrectAuth?.headerName, incorrectAuth?.headerValue),
+  });
+
+  it('returns a 200 response with the activation transaction if it is returned by ACBS', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    givenRequestToGetFacilityActivationTransactionInAcbsSucceeds();
+
+    const { status, body } = await api.get(getFacilityActivationTransactionUrl);
+
+    expect(status).toBe(200);
+    expect(body).toStrictEqual(apiFacilityActivationTransaction);
+  });
+
+  it('returns a 404 response if ACBS returns a 400 response with the string "BundleInformation not found"', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    requestToGetFacilityActivationTransaction().reply(400, 'BundleInformation not found or user does not have access to it.');
+
+    const { status, body } = await api.get(getFacilityActivationTransactionUrl);
+
+    expect(status).toBe(404);
+    expect(body).toStrictEqual({
+      statusCode: 404,
+      message: 'Not found',
+    });
+  });
+
+  it('returns a 500 response if ACBS returns a 400 response without the string "BundleInformation not found"', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    requestToGetFacilityActivationTransaction().reply(400, 'An error message from ACBS.');
+
+    const { status, body } = await api.get(getFacilityActivationTransactionUrl);
+
+    expect(status).toBe(500);
+    expect(body).toStrictEqual({
+      statusCode: 500,
+      message: 'Internal server error',
+    });
+  });
+
+  it('returns a 500 response if ACBS returns a 400 response that is not a string when getting the activation transaction', async () => {
+    const acbsErrorMessage = { Message: 'error message' };
+    givenAuthenticationWithTheIdpSucceeds();
+    requestToGetFacilityActivationTransaction().reply(400, acbsErrorMessage);
+
+    const { status, body } = await api.get(getFacilityActivationTransactionUrl);
+
+    expect(status).toBe(500);
+    expect(body).toStrictEqual({
+      statusCode: 500,
+      message: 'Internal server error',
+    });
+  });
+
+  it('returns a 500 response if getting the loan transaction from ACBS returns a status code that is NOT 200 or 400', async () => {
+    givenAuthenticationWithTheIdpSucceeds();
+    requestToGetFacilityActivationTransaction().reply(401);
+
+    const { status, body } = await api.get(getFacilityActivationTransactionUrl);
+
+    expect(status).toBe(500);
+    expect(body).toStrictEqual({
+      statusCode: 500,
+      message: 'Internal server error',
+    });
+  });
+
+  it('returns a 500 response if getting the loan transaction from ACBS times out', async () => {
```

Comment:
Should we have a 400 test for if the 1st item of the returned bundle isn't a facility code value transaction?